src/Template/Bake/Element/form.twig

```diff
@@ -0,0 +1,38 @@
+<div class="{{ pluralVar }} form content">
+    <?= $this->Form->create(${{ singularVar }}) ?>
+    <fieldset>
+        <legend><?= __('{{ action|humanize }} {{ singularHumanName }}') ?></legend>
+        <?php
+{% for field in fields if field not in primaryKey %}
+    {%- if keyFields[field] %}
+        {%- set fieldData = Bake.columnData(field, schema) %}
+        {%- if fieldData.null %}
```

Comment:
Is `fieldData.null` valid usage?

Reply:
I'm not an expert in twig. So i took it from [cakephp/bake repo](https://github.com/cakephp/bake/blob/master/src/Template/Bake/Element/form.twig#L51).

Reply:
Hmm I made those templates but don't remember :P